manufacturing-ai-suite/industrial-edge-insights-multimodal/docs/user-guide/weld-defect-detection/get-started.md

```diff
@@ -0,0 +1,240 @@
+# Get Started
```

Comment:
I think we can delete this as this comes at root level as can be seen at https://github.com/open-edge-platform/edge-ai-suites/tree/release-2025.2.0/manufacturing-ai-suite/industrial-edge-insights-time-series/docs/user-guide/weld-anomaly-detection